models/scan.go

```diff
@@ -0,0 +1,43 @@
+package models
+
+import "github.com/greenbone/eulabeia/messages"
+
+// Scan contains Target as well as volatile information for a specific scan
+type Scan struct {
+	Target
+	ID      string   `json:"id"`      // ID of a Scan
+	Exclude []string `json:"exclude"` // Exclude hosts from scan
```

Comment:
why is excluded part of scan, and not part of target?

Reply:
Actually we need both; Scan for start and stop and within Target to exclude hosts from a given range. That will be changed in scan.finished and target.exclude; initially I thought exclude was just used for start, stop, resume hence on scan.

Reply:
Resolved by: f394486000c84714fe14693462a9b33593109dda